src/Weather/Rasp/RaspStyle.cpp

```diff
@@ -116,8 +116,312 @@ static constexpr ColorRamp rasp_colors[6][NUM_COLOR_RAMP_LEVELS] = {
     { 300, { 255, 153, 0 }},
     {1000, { 255, 102, 0 }},
   },
+
+  // Thermalmap.info Color schemes
```

Comment:
Instead of adding more array entries here, please create new variables with expressive names.